cassandra-mesos-model/src/main/java/io/mesosphere/mesos/util/CassandraFrameworkProtosUtils.java

```diff
@@ -172,6 +180,83 @@ public static CassandraNodeTask getTaskForNode(@NotNull final CassandraNode cass
         return builder;
     }
 
+    @NotNull
+    public static Function<Resource, TreeSet<Long>> resourceToPortSet() {
+        return new Function<Resource, TreeSet<Long>>() {
```

Comment:
Please make a private static final inner class of this function with a singleton instance returned by this method.

See https://github.com/mesosphere/cassandra-mesos/blob/master/cassandra-mesos-model/src/main/java/io/mesosphere/mesos/util/CassandraFrameworkProtosUtils.java#L176 for an example of how this had been done previously.